omod/src/main/java/org/openmrs/module/idgen/rest/search/LogEntrySearchHandler.java

```diff
@@ -58,13 +58,19 @@ public PageableResult search(RequestContext context) throws ResponseException {
         Date dateFrom = fromDate != null ? (Date) ConversionUtil.convert(fromDate, Date.class) : null;
         Date dateTo = toDate != null ? (Date) ConversionUtil.convert(toDate, Date.class) : null;
         User user = generatedBy != null ? service.getUserByUuid(generatedBy): null;
-        if (logSource == null && identifier == null && comment == null && dateFrom == null && dateTo == null
-                && user == null) {
+        if (source != null && logSource == null) {
             return new EmptySearchResult();
         }
-        List<LogEntry> logEntries = identifierSourceService.getLogEntries(logSource, dateFrom, dateTo, identifier, user, comment);
 
-        return new NeedsPaging<LogEntry>(logEntries, context);
+        else if (generatedBy != null && user == null) {
+            return new EmptySearchResult();
+        } 
+        
```

Comment:
What is the use of these empty lines?

Reply:
On one of my former PRs , one of the developers recommended that I should leave an empty line between code blocks

Reply:
Empty lines between if and elses???